t/066-socket-receiveuntil.t

```diff
@@ -126,7 +126,7 @@ GET /t
 qq{connected: 1
 request sent: 57
 read: HTTP/1.1 200 OK
-read: Server: nginx
+read: Server: openresty
```

Comment:
This is not necessary.
The CI did apply the patch for the server name.

Reply:
got it